packages/SwingSet/src/kernel/vatAdmin/vatAdminWrapper.js

```diff
@@ -10,38 +10,39 @@ import { producePromise } from '@agoric/produce-promise';
 
 export default function setup(syscall, state, helpers) {
   function build(E, D) {
-    const vatIdsToResolvers = new Map();
+    const pendingVatIDs = new Map();
```

Comment:
naming a map for the keys rather than the values is unusual.

Reply:
what about just `pending` ?

Reply:
done